CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# w3c/vc-di-ecdsa-test-suite  ChangeLog
+
+## 1.0.0
```

Comment:
Are we going to include changelogs in all test suites moving forward? and where are we planning to publish them?
```suggestion
## 1.0.0 - 2023-11-dd
```